examples/config.example.json

```diff
@@ -1,18 +1,32 @@
 {
     "work_dir": "/absolute/path/to/work_dir",
 
-    "google_play_accounts": {
+    "products": {
         "aurora": {
+            "has_nightly_track": false,
```

Comment:
I wonder if we could make this piece of config even more generic. For instance:
```json
"additional_tracks": ["nightly"]
```

How does that sound?

Reply:
Okay. Sounds good to me!